android/src/main/java/io/radar/react/RNRadarModule.java

```diff
@@ -87,16 +87,24 @@ public void getPermissionsStatus(Promise promise) {
             return;
         }
 
-        boolean foreground = ActivityCompat.checkSelfPermission(getReactApplicationContext(), Manifest.permission.ACCESS_FINE_LOCATION) == PackageManager.PERMISSION_GRANTED;
+        Activity activity = getCurrentActivity();
+
+        boolean foreground = ActivityCompat.checkSelfPermission(activity, Manifest.permission.ACCESS_FINE_LOCATION) == PackageManager.PERMISSION_GRANTED;
+        boolean background = true;
```

Comment:
There was no separate background permission before API level 29, this flag is updated below

Reply:
should this be `boolean background = foreground;`?  Otherwise on VERSION < 29 it's always = `true` and we'll always `promise.resolve("GRANTED_BACKGROUND");`

Reply:
That's the intended behavior since there was no separate background permission before API level 29

Reply:
if you denied foreground perms, you would still have background perms?

Reply:
Ohhh I see what you're saying now. Will update

Reply:
Good catch, done